src/components/slider/slider.tsx

```diff
@@ -175,7 +176,7 @@ export class Slider {
                         aria-valuenow={this.multiplyByFactor(this.getValue())}
                         aria-label={this.label}
                         aria-disabled={this.disabled || this.readonly}
-                        data-step={this.step}
+                        data-step={this.multiplyByFactor(this.step)}
```

Comment:
Just for the record:

I found this little visual bug when testing in the web client:

https://user-images.githubusercontent.com/154725/119490879-9dae8300-bd5d-11eb-94ec-e8dd01e6a3b3.mov

The underlying value is still just `0`, the value can be saved, and once the card is loaded again, it shows as `0`, as expected. The bug is there before this change too, but it seems it's easier to trigger when the `step` property is used. Without this fix (and the corresponding fix in lime-crm-components), the width of the area that triggers the bug is only 1px wide. With this fix, and the `step` set to 1/10 of the max value, the bug is triggered by an area approximately 10-15px wide.